src/wrapper/java_vm/vm.rs

```diff
@@ -130,6 +130,7 @@ use crate::InitArgs;
 /// [actp]: struct.JavaVM.html#method.attach_current_thread_permanently
 /// [actd]: struct.JavaVM.html#method.attach_current_thread_as_daemon
 /// [spec-references]: https://docs.oracle.com/en/java/javase/12/docs/specs/jni/design.html#referencing-java-objects
+#[repr(transparent)]
```

Comment:
Makes sense. Doc for reviewers: https://doc.rust-lang.org/1.26.2/unstable-book/language-features/repr-transparent.html